app/controllers/api/users.php

```diff
@@ -30,19 +31,20 @@
     ->label('sdk.response.code', Response::STATUS_CODE_CREATED)
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_USER)
+    ->param('id', 'uid', new Key(), 'Unique Id. Passing `uid()` will auto generate the id.')
```

Comment:
```suggestion
    ->param('userId', 'unique()', new Key(), 'Unique Id. Passing the string `unique()` will auto generate your ID.')
```
- `userId` is more consistent
- `unique()` feels more verbose, we need to adjust Key to accept it.

Reply:
I cannot actually use `unique()` the `()` will cause the validator to fail?

Reply:
Why `userId` instead of just `id`? 🤔